db_stress_tool/db_stress_test_base.cc

```diff
@@ -1725,17 +1741,28 @@ void StressTest::Open() {
           break;
         }
       }
+      assert(s.ok());
 #else
       fprintf(stderr, "Secondary is not supported in RocksDBLite\n");
       exit(1);
 #endif
     }
+    if (FLAGS_continuous_verification_interval > 0 && !cmp_db_) {
+      Options tmp_opts;
+      tmp_opts.max_open_files = FLAGS_open_files;
```

Comment:
Should we use -1?

Reply:
For now, yes. I will change it to -1 and add a TODO.